monero-wallet-gui.pro

```diff
@@ -286,7 +286,9 @@ win32 {
         -lsetupapi \
         -lssl \
         -lsodium \
-        -lcrypto
+        -llibprotobuf \
```

Comment:
```suggestion
```

Reply:
ping @mmbyday

Reply:
Nice catch. Local build works fine without it.